boilerplates/mantine/files/components/$Link.tsx.ts

```diff
@@ -0,0 +1,14 @@
+export default async function overrideLayout() {
+  return `import React from "react";
+import { usePageContext } from "vike-react/usePageContext";
+import { NavLink } from '@mantine/core';
+
+export function Link({ href, label }: { href: string; label: string }) {
+  const pageContext = usePageContext();
+  const { urlPathname } = pageContext;
+  const isActive = href === "/" ? urlPathname === href : urlPathname.startsWith(href);
+  return (
+    <NavLink href={href} label={label} active={isActive} />
+  );
+}`;
+}
```

Comment:
Usually, for simple modifications like this one, you'd want to make use of [tsx related templating syntax](https://github.com/vikejs/bati/blob/main/boilerplates/README.md) directly in the original file.